addon/mixins/ajax-request.js

```diff
@@ -615,7 +616,7 @@ export default Mixin.create({
    */
   generateDetailedMessage(status, headers, payload, requestData) {
     let shortenedPayload;
-    const payloadContentType = headers['Content-Type'] || 'Empty Content-Type';
+    const payloadContentType = getHeader(headers, 'Content-Type') || 'Empty Content-Type';
```

Comment:
There isn't really a default. The standard in case the type is unknown to the sender is to [omit the Content-Type header entirely](http://stackoverflow.com/questions/1176022/unknown-file-type-mime/28652339#28652339), so the absence of a header is a valid "value" in its own right.

The string 'Empty Content-Type' just gets placed in a friendly error description field anyway, so I think it's fine. Unless we want to change the message to a more explicit 'No Content-Type Header' or somesuch; I'd be down with that.